Apps/AdminWebClient/src/ClientApp/src/views/Dashboard.vue

```diff
@@ -94,5 +104,11 @@ export default class Dashboard extends Vue {
       this.unregisteredInvitedUserCount = count;
     });
   }
+
+  private getUsersWithNotesCount() {
```

Comment:
I think this should pass a parameter for the count of notes instead of getting any count